### PR TITLE
docs: grammar updates to en-us guides

### PR DESCRIPTION
**Pull request by jameesy**
## PR Checklist

- [x] Fix linting errors
- [x] Label has been added

## Change information
I read through the en-us guides and have updated/fixed some of the grammar.

### CODE REVIEW

File: pages/en-us/guide/server-render.mdx [hunk 1]
```diff
@@ -15,21 +15,20 @@ export const meta = {
...
 <Spacer y={.5} />
 
-Will using server-side rendering make my application lose the advantages of SPA? Maybe you can use the `hybrid render` application,
-Read <Link color href="https://nextjs.org/blog/next-9-3#next-gen-static-site-generation-ssg-support" rel="nofollow">
+If you are concerned about losing the advantages of a Single Page Application by implementing server-side rendering, you can use the `hybrid render` application.
+Plese read <Link color href="https://nextjs.org/blog/next-9-3#next-gen-static-site-generation-ssg-support" rel="nofollow">
 the post from the Next.js team</Link> to learn more.
 
 <Text>In addition, for <Text b>server-side render</Text> and web applications,
 we strongly recommend that you read this famous post
```
Comment (unix) [inline pages/en-us/guide/server-render.mdx:29]: Spaces between `post` and `7-` are ignored after compilation.

Comment (unix) [reply pages/en-us/guide/server-render.mdx:29]: You can see the difference here: [preview](https://react-q2fkaviap.vercel.app/en-us/guide/server-render)

Comment (jameesy) [reply pages/en-us/guide/server-render.mdx]: Fixed

**Comment by jameesy** [comment]
> Can I merge it now?

Sure thing! Thanks 🙂

**Comment by unix** [comment]
@jameesy 👏 👏 
Thank you for make `zeit-ui` better.

**Pull request merged**